app/Models/WaitingRoom.php

```diff
@@ -29,6 +29,19 @@ public function users()
         return $this->belongsToMany(User::class);
     }
 
+    /**
+     * Determines if the waiting room is open for signupsOpen
+     *
+     * @return bool whether the room is open or not
+     */
+    public function isOpen()
+    {
+        $start = Carbon::parse($this->signup_start_date)->startOfDay();
```

Comment:
They're already casted so that should work :)

```
// Models/WaitingRooms.php
/**
     * The attributes that should be mutated to dates.
     *
     * @var array
     */
    protected $dates = ['signup_start_date', 'signup_end_date'];
```